packages/react/src/components/RadioTile/RadioTile.tsx

```diff
@@ -5,21 +5,33 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import React, { forwardRef, ChangeEvent, KeyboardEvent, ReactNode, Ref } from 'react';
 import { CheckmarkFilled } from '@carbon/icons-react';
 import cx from 'classnames';
 import PropTypes from 'prop-types';
-import React from 'react';
 import { keys, matches } from '../../internal/keyboard';
 import { useFallbackId } from '../../internal/useId';
 import { usePrefix } from '../../internal/usePrefix';
 import deprecate from '../../prop-types/deprecate';
 
-const RadioTile = React.forwardRef(function RadioTile(
+interface RadioTileProps {
+  checked: boolean;
+  children: ReactNode;
+  className?: string;
+  disabled: boolean;
+  id?: string;
+  light: boolean; // You may update this to match the actual type
```

Comment:
Please include a `@deprecated` typescript comment/annotation on this property with an identical message as what's below in proptypes.